superset/annotation_layers/annotations/commands/exceptions.py

```diff
@@ -68,5 +64,5 @@ class AnnotationUpdateFailedError(CreateFailedError):
     message = _("Annotation could not be updated.")
 
 
-class AnnotationDeleteFailedError(CommandException):
-    message = _("Annotation delete failed.")
+class AnnotationDeleteFailedError(DeleteFailedError):
+    message = _("Annotations could not be deleted.")
```

Comment:
Rather than `chart(s)`, `query(ies)`, or `query/queries` I opted for—per a Google Search suggestion—always using the plural form for consistency.